src/components/AccessibilityRequestsTable/index.test.tsx

```diff
@@ -20,6 +49,25 @@ describe('AccessibilityRequestsTable', () => {
     expect(wrapper.find('th').at(4).contains('Status')).toBe(true);
   });
 
+  it('displays relevant results from filter', async () => {
+    render(
+      <MemoryRouter>
+        <AccessibilityRequestsTable requests={requests} />
```

Comment:
This type error is mainly due to the mock `requests` object not having the `__typename` properties that the GraphQL types have. There's also an issue with `.statusRecord.status` being a raw string instead of the enum defined in graphql-global-types.ts. Possible solutions:

1. Ignore the TS errors for this PR.
2. Add the `__typename` fields and change the statuses to enum values (i.e. `status: AccessibilityRequestStatus.IN_REMEDIATION`) in the mock data in this test file.
3. Try and wrangle the TypeScript types to exclude the `__typename` fields. I don't think this is a good idea for this PR, it's a nontrivial amount of work.

Up to you how you want to proceed. If you want to go with 2, I've got the necessary changes locally, I can commit and push them to your branch if you want.

Reply:
Yeah I ran into that existing ts error, much appreciated if you want to push your local changes to my branch.  Thanks Dylan

Reply:
Just pushed my changes to fix the type error.